src/assets/icons/bell.tsx

```diff
@@ -1,3 +1,13 @@
+import { DetailedHTMLProps, FC, SVGProps } from 'react';
+
+
+export const BellIcon: FC<
+  DetailedHTMLProps<SVGProps<SVGSVGElement>, SVGSVGElement>
+> = () => (
+
```

Comment:
why are all these svg files tsx???

Reply:
I wanted to do it like we did in nine-mobile